Doc/glossary.rst

```diff
@@ -451,8 +451,10 @@ Glossary
       for best practices on working with annotations.
 
    __future__
-      A pseudo-module which programmers can use to enable new language features
-      which are not compatible with the current interpreter.
+      A :ref:`future statement <future>`, "from __future__ import *feature*
+      ...", directs the compiler to compile the current module using syntax or
```

Comment:
Code can use three backticks ` rather than quote marks

Reply:
I will look at the result either way.  I will also move the paragraph break and remove the duplicate module reference that follows.

Reply:
Two backticks, not one or three, are needed for inline code in .rst.  I decided to remove ` ...` from the quoted code; one interested in the full syntax will have to look at the doc.  I also removed the paragraph break.

Reply:
I changed `*feature*` to `<feature>` as the latter is, I believe, used more for a syntax variable.